api/node/__test__/js_value_conversion.spec.ts

```diff
@@ -237,6 +237,8 @@ test('get/set brush properties', (t) => {
     in-out property <brush> black: #000000;
     in-out property <brush> trans: transparent;
     in-out property <brush> ref: transparent;
+    in-out property <brush> linerar-gradient: @linear-gradient(90deg, #3f87a6 0%, #ebf8e1 50%, #f69d3c 100%);
```

Comment:
`linerar` -> `linear` (here and in the test below)

Reply:
done